qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -381,7 +383,8 @@ def test_get_without_target_name(self):
 
         # get dump5 should fail
         o = self.get_cephfs_shell_cmd_output("get dump5")
-        o = self.get_cephfs_shell_cmd_output("!stat dump5 || echo $?")
+        # cwd=None because we want to run it at CWD, not at cephfs mntpt.
+        o = self.mount_a.run_shell('stat dump5 || echo $?', cwd=None).stdout.getvalue()
```

Comment:
This would fail as :

```
stat: cannot statx '||': No such file or directory
stat: cannot statx 'echo': No such file or directory
stat: cannot statx '$?': No such file or directory
```

Maybe `Raw` can be used to make the shell treat the whole command as one or `self.mount_a.run_shell('stat dump5', cwd=None, check_status=True).returncode` can be used for assertion?